examples/podinfo/examples/podinfo.ts

```diff
@@ -1,5 +1,5 @@
 import { Deployment, ImagePullPolicy, IContainer } from "../lib";
-import { Container } from '../../imports/k8s';
+import { Container } from '../.gen/apps-deployment-v1'
```

Comment:
New way of importing was not working for me. Does https://github.com/awslabs/cdk8s/commit/2db4cfbb20e7b6d8b09e8b8c9e6d096f0ea9fefe need to be published?

Reply:
Yes, try to install the @next dist tag:

“yarn global add cdk8s-cli@next”
“yarn add cdk8s@next”

Reply:
Awesome, that worked!